praw/models/reddit/subreddit.py

```diff
@@ -306,6 +308,22 @@ def wiki(self):
             self._wiki = SubredditWiki(self)
         return self._wiki
 
+    @property
+    def emoji(self):
+        """Provide an instance of :class:`.SubredditEmoji`.
+
+        This attribute can be used to discover all emoji for a subreddit:
+
+        .. code:: python
+
+           for emoji in reddit.subreddit('iama').emoji:
```

Comment:
I thought about commenting on this also. Wikipages use `__iter__` instead of `__call__`. The reason many use `__call__` is so that arguments can be passed into the iterator, such as a filter, or a limit. I'm okay with this remaining `__iter__` for now because emoji listings don't appear to take any parameters. If in the future parameters are needed we can transition to `__call__` and deprecate `__iter__`.

I think using `__iter__` in combination with the `emoji['name']` syntax makes sense since it's like we're working with a dictionary.